src/lib/bootloader/bootloader_base.rb

```diff
@@ -74,5 +80,29 @@ def merge(other)
       @read ||= other.read?
       @proposed ||= other.proposed?
     end
+
+  private
+
+    # @return [Boolean] true when kexec-tools package should be included; false otherwise
+    def include_kexec_tools_package?
+      return false if Yast::Mode.live_installation
+
+      Yast::Linuxrc.InstallInf("kexec_reboot") != "0"
+    end
+
+    # done in common write but also in installation pre write as kernel update need it
+    # @param prewrite [Boolean] true only in installation when scr is not yet switched
+    def write_sysconfig(prewrite: false)
```

Comment:
why it is moved to private? This cause exception during installation at https://github.com/yast/yast-bootloader/blob/master/src/clients/inst_bootloader.rb#L45

Reply:
Sorry, I did not realize about it.